frontend/src/classes/Campaign.ts

```diff
@@ -13,10 +13,25 @@ export enum Status {
   Halted = 'Halted',
 }
 
+export enum StatusFilter {
```

Comment:
hmm I see you're extending the existing style, but these are the same enums repeated in `backend/src/core/constants.ts` and should really be extracted out to the shared folder too hmm (maybe we can leave a comment here for future refactoring, instead of dealing with them in the same PR)

Reply:
Future refactoring